api-reference/webhooks.mdx

```diff
@@ -62,54 +62,55 @@ the signature is made out of 2 components:
 <CodeGroup>
 
 ```javascript javacript
```

Comment:
do u need second javascript def? probly first should do it

Reply:
yes- first one is syntax highlighting, second is mintlify tab def